src/models/Variable.js

```diff
@@ -0,0 +1,55 @@
+import FileInformation from './FileInformation';
+
+class Variable extends FileInformation {
```

Comment:
Missing documentation

Reply:
Done.

Reply:
`Class that represents a variable in a file of a model.`

Reply:
Done.